src/lock-bot.ts

```diff
@@ -25,6 +25,7 @@ export type Destination = "user" | "channel";
 export interface Response {
   message: string;
   destination: Destination;
+  metaData?: Record<string, string>;
```

Comment:
This `metaData` was never meant to be read out of the DynamoDB.
I don't want us to read the `metaData` out of the DB ever and maybe we should just pass an optional `description?: string` back in the `Response`.

That wasn't for you to know, it's not obvious, I thought you added it at first 😅 
The `metaData` is just to be able to store human-readable data in the database for the rare occasion I need to look in there and see how people are using it - to understand current use-cases. Otherwise it's a lot of unreadable team/channel/user slack codes.